app.py

```diff
@@ -1,88 +1,135 @@
 #!/usr/bin/python3
 
-import argparse
 import logging
+import os
+import sys
 
-from flask import Flask, render_template
+from flask import Flask, render_template, abort
+from flask import jsonify
 
-from db.dao.assignments_dao import AssignmentDao
-from db.dao.student_dao import StudentDao
-from db.dao.task_dao import TaskDao
-from db.database import ClassroomDb
+sys.path.append(os.getcwd())
 
-"""COMMAND LINE ARGUMENTS"""
+"""APP CONFIGURATION """
+ip = '127.0.0.1'
+port = 5000
 
-
-def get_arguments():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--ip', dest='ip', default='127.0.0.1',
-                        help='Optional. IP address to bind the server to. Default is 127.0.0.1')
-    parser.add_argument('-p', '--port', dest='port', default=5000,
```

Comment:
it turns out that those cli properties can override flask cli properties, so i decided to remove them